### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by shihabrafio**
This PR includes,

-   The concept of state management.
-   Use store, actions and reducers in React.
-   Connect React and Redux.
-   Apply React best practices and language style guides in code.

### CODE REVIEW

File: src/components/Addbook.jsx [hunk 1]
```diff
@@ -73,15 +73,15 @@ const Div = styled.div`
     .title {
       height: 2.813rem;
       border-radius: 4px;
-      border: solid 1px #2e90f0;
+      border: solid 1px #e1e1e1;
```
Comment (alexander16108) [inline src/components/Addbook.jsx:77]: - as seen below there should be some padding between the cards and the navbar
- as seen below the background of your books are different from the original design please fix this 👍🏾 
- your  book should have padding around the body of the book as seen below
- please remove the border around the see progress button 
- please use the correct font for this project which is `Roboto, Helvetica, sans-serif

your design
image

required design
image

File: src/components/Addbook.jsx [hunk 2]
```diff
@@ -73,15 +73,15 @@ const Div = styled.div`
...
       background-color: #fff;
       padding-left: 1rem;
       width: 55.15%;
     }
     .author {
       height: 2.813rem;
       border-radius: 4px;
-      border: solid 1px #2e90f0;
+      border: solid 1px #e1e1e1;
```
Comment (alexander16108) [inline src/components/Addbook.jsx:85]: - your nav links should be correctly aligned to the centre with the header and user icon as seen below 👍🏾 
- the colour of the header which should be blue Is different from the original design please fix this error 👍🏾
- the font size of your nava header should match the original design and padding as well  please fix this 👍🏾 
- the color of your link should be black please update this  
- the user icon should have some significant padding to the right to proper resemble the original design 

required design 
image

your design 
image

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @shihabrafio , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @shihabrafio, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- Styled Bookstore using correct padding, font style, button style, and font weight. ✔️ 
 
## [Optional] suggestions:
* For future pull requests, please notice that your branch name `style`does not follow the convention of being named after a feature or action. For example, you could name your branch `added-style`. This makes it easier to track the changes and the purpose of each branch.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**